CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+### 0.4.3
+
+Released 2019/02/18.
+
+* `wasm32` intrinsics are now invoked using `core::arch` rather than LLVM.
```

Comment:
```suggestion
* Add support for building on stable Rust on Windows and Unix.
* `wasm32` intrinsics are now invoked using `core::arch` rather than LLVM.
```

Reply:
Thanks!